packages/styles/notice.css

```
@@ -80,7 +98,16 @@
   color: var(--notice-link-hover-color);
 }
 
-.Notice .Notice__content > p:first-of-type {
+.Notice .Notice__content-wrapper > p:first-of-type {
```

I think we can remove this class now.

```suggestion
.Notice .Notice__content-wrapper > p:first-of-type {
```